libbeat/scripts/Makefile

```diff
@@ -457,7 +457,7 @@ package-setup:
 
 .PHONY: package
 package: ## @packaging Create binary packages for the beat.
-package: package-setup
```

Comment:
I wonder if we should even run `clean` before `update` to make sure everything is "fresh".